examples/Huggingface_Transformers/README.md

```diff
@@ -51,21 +51,25 @@ In the setup_config.json :
 
 *embedding_name* : The name of embedding layer in the chosen model, this could be `bert` for `bert-base-uncased`, `roberta` for `roberta-base` or `roberta` for `xlm-roberta-large`, or `gpt2` for `gpt2` model
 
+*hardware* : The target platform to trace the model for. Specify as `neuron` for Inferentia1.
+
+*batch_size* : Input batch size when tracing the model for `neuron` as target hardware.
```

Comment:
Is this applicable only to Inferentia?

Diff looks good overall.

Nit: Please mention  a sentence or two about Inferentia or point to a url with details.

Also, do we want to mention about this in the benchmark ReadMe

Reply:
Yes, that's correct. Currently `batch_size` only affects model tracing for Inferentia.
Will update docs to include reference to Inferentia.